markdown/summary.Rmd

```diff
@@ -6,6 +6,33 @@ output:
 
 ```{r, echo=FALSE, message=FALSE, warning=FALSE,error=FALSE}
 
+# Source setup environment and function scripts
+source(here::here("code", "00_setup-environment.R"))
```

Comment:
I think souce() here does work for me. If I open summary.Rmd and knit it, it will create "data" folder within "markdown" folder, and pop a message saying "Failed to search directory 'markdown'".  I have a feeling it's because summary.Rmd is in the markdown folder. When running Create data and figures folders in 00_setup-environment, it will look for the folders "data" and "markdown" within "markdown" folder. Will look into how to fix it.

Reply:
Thanks Tina - I looked at doing this but this would mean updating dates in multiple places for each run of the publication. I've used the here package and think this now works well.